pkg/installation/install.go

```diff
@@ -94,5 +106,62 @@ func Remove(p environment.KrewPaths, name string) error {
 	}
 	glog.V(1).Infof("Deleting plugin version %s", version)
 	glog.V(3).Infof("Deleting path %q", filepath.Join(p.Install, name))
+	removeLink(p.Bin, name)
 	return os.RemoveAll(filepath.Join(p.Install, name))
 }
+
+func createOrUpdateLink(binDir string, binary string, plugin string) error {
+	dst := filepath.Join(binDir, pluginNameToBin(plugin))
+
+	if err := removeLink(binDir, plugin); err != nil {
+		return fmt.Errorf("failed to remove old symlink, err: %v", err)
+	}
+	if _, err := os.Stat(binary); os.IsNotExist(err) {
+		return fmt.Errorf("can't create file, destination %q does not exsist", binary)
+	}
+
+	// Create new
+	glog.V(2).Infof("Creating symlink from %q to %q\n", binary, dst)
+	if err := os.Symlink(binary, dst); err != nil {
+		return fmt.Errorf("failed to create a symlink form %q to %q, err: %v", binDir, dst, err)
+	}
+	glog.V(2).Infof("Created symlink in %q\n", dst)
+
+	return nil
+}
+
+func checkSubPath(subPath, path string) error {
+	subPathAbs, err := filepath.Abs(subPath)
+	if err != nil {
+		return fmt.Errorf("failed to get the absolute path of %q, err: %v", subPath, err)
+	}
+	pathAbs, err := filepath.Abs(path)
+	if err != nil {
+		return fmt.Errorf("failed to get the absolute path of %q, err: %v", path, err)
+	}
+	if _, ok := pathutil.IsSubPath(subPathAbs, pathAbs); !ok {
+		return fmt.Errorf("the path %q does not extend the sub-path %q", path, subPath)
+	}
+	return nil
+}
+
+func removeLink(binDir string, plugin string) error {
+	dst := filepath.Join(binDir, pluginNameToBin(plugin))
+	if err := checkSubPath(binDir, dst); err != nil {
+		return fmt.Errorf("failed to remove link, err: %v", err)
+	}
+	if err := os.Remove(dst); err != nil && !os.IsNotExist(err) {
+		return fmt.Errorf("failed to remove the symlink in %q, err: %v", dst, err)
+	} else if err == nil {
+		glog.V(3).Infof("Removed symlink from %q\n", dst)
```

Comment:
nit:  trailing `\n` is  redundant. same comment for line 124 and 128:

	124: glog.V(2).Infof("Creating symlink from %q to %q\n", binary, dst)
	128: glog.V(2).Infof("Created symlink in %q\n", dst)